department-of-veterans-affairs/va.gov-cms
# VAGOV-000: Various Sprint 15 AX improvements.

This PR targets a number of very small user stories that all affect form display config, so they are rolled together here to reduce conflicts and avoid lag times. 
![VAGOV_board_-_Agile_Board_-_Jira](https://user-images.githubusercontent.com/643678/58269589-e50c6e00-7d55-11e9-8396-eaa0721a89c8.jpg)

**QA notes:**
Paragraphs stuff: https://va-gov.atlassian.net/browse/VAGOV-3216
Intro/Description field: https://va-gov.atlassian.net/browse/VAGOV-2514
Meta title: https://va-gov.atlassian.net/browse/VAGOV-2727
Metatags: https://va-gov.atlassian.net/browse/VAGOV-3366
Reusable alerts and featured content block on detail page: https://va-gov.atlassian.net/browse/VAGOV-3779
Leadership page intro text: https://va-gov.atlassian.net/browse/VAGOV-3719
Press releases intro text: https://va-gov.atlassian.net/browse/VAGOV-3720

All content model changes are additive (new fields) or superficial (changing field labels), therefore should not break any build.
